marlowe-playground-client/src/BottomPanel/Types.purs

```diff
@@ -44,3 +46,32 @@ _showBottomPanel = prop (SProxy :: SProxy "showBottomPanel")
 
 _panelView :: forall panel. Lens' (State panel) panel
 _panelView = prop (SProxy :: SProxy "panelView")
+
+class ShowConstructor a where
```

Comment:
The BottomPanel is an abstract UI component that knows nothing about it's content and could/should be moved to the web-commons. Please move the MetadataAction to a Metadata specific module